python/huggingface_server.Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG POETRY_HOME=/opt/poetry
 ARG POETRY_VERSION=1.7.1
 
 # Install vllm
-ARG VLLM_VERSION=0.4.2
+ARG VLLM_VERSION=0.4.3
```

Comment:
Any reason to upgrade to 0.4.3 ? seems like there are some breaking changes which affects us.